src/views/components/AnalyticsGraph/index.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <div :id="this.title">
+    <svg width="80vw" height='33vh'></svg>
+  </div>
+</template>
+<script>
+import * as d3 from "d3";
+export default {
+  name: 'graph',
+  data () {
+    return {
+    };
+  },
+  props : ['values' , 'title', 'xUnits', 'yUnits'],
+
+  mounted() {
+    /**@description Create an event listener for window resizing that rerenders the graph */
```

Comment:
so function comments are usually like this! 
`/**
 * Creates blah blah blah
 * @returns array of stuff
 */`

it'd be great if we could get the linting to work hahah

Reply:
```
/**
*  Creates blah blah blah
*
* @returns {string[]} The array of blah blah blah
*/
```